### PR TITLE
[QPE-367] verification issues 

### PR DESCRIPTION
**Pull request by ahmed-Bazzara**
* removing all sorts of console logs

### CODE REVIEW

**Comment by ahmed-Bazzara** [comment]
https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-367

File: src/utilities.js [hunk 1]
```diff
@@ -56,13 +56,6 @@ export function addOnActivateButtonEvent ($element,config,layout,url,recipient,t
       textboxReference.addEventListener('click', function() {
         var copyTextarea = document.querySelector('.linkTextboxArea');
         copyTextarea.select();
-        try {
-          var successful = document.execCommand('copy');
```
Comment (kristofferlind) [inline src/utilities.js]: `document.execCommand('copy');` is what's actually copying to clipboard, pretty sure you need to leave it in there.

File: src/utilities.js [hunk 2]
```diff
@@ -122,12 +115,5 @@ function copyTextToClipboard(text) {
 
   textArea.select();
 
-  try {
-    var successful = document.execCommand('copy');
```
Comment (kristofferlind) [inline src/utilities.js]: as above

File: src/utilities.js (hunk 1 above)
Comment (ahmed-Bazzara) [reply src/utilities.js]: aha
i thought it only was for the sake of logging a successful/unsuccessful message

File: src/utilities.js [hunk 3]
```diff
@@ -57,11 +57,10 @@ export function addOnActivateButtonEvent ($element,config,layout,url,recipient,t
...
         try {
-          var successful = document.execCommand('copy');
-          var msg = successful ? 'successful' : 'unsuccessful';
-          console.log('Copying text command was ' + msg);
-        } catch (err) {
-          console.log('Oops, unable to copy.');
+          document.execCommand('copy');
+        }
+        catch (err) {
+          console.log(err); /*eslint no-console: 0*/
```
Comment (kristofferlind) [inline src/utilities.js]:
```suggestion
          console.log(err); // eslint-disable-line no-console
```

File: src/utilities.js [hunk 4]
```diff
@@ -121,13 +120,11 @@ function copyTextToClipboard(text) {
...
   textArea.select();
-
   try {
-    var successful = document.execCommand('copy');
-    var msg = successful ? 'successful' : 'unsuccessful';
-    console.log('Copying text command was ' + msg);
+    document.execCommand('copy');
   } catch (err) {
-    console.log('Oops, unable to copy');
+    console.log(err); /*eslint no-console: 0*/
```
Comment (kristofferlind) [inline src/utilities.js]:
```suggestion
    console.log(err); // eslint-disable-line no-console
```

**Pull request merged**